experimental/PropertyDDS/packages/property-inspector-table/tsconfig.json

```diff
@@ -1,20 +1,22 @@
 {
   "extends": "@fluidframework/build-common/ts-common-config.json",
   "compilerOptions": {
+    "module": "esnext",
```

Comment:
if your using es-next for your regular build, there is no need for having both an esnext build and another build. This 2 build pattern only really makes sense if you want to include both a commonJS modules and an esnext (aka ESM) build in the package.

Reply:
Yes, but that the intention to have both versions in the package.